socialhome/streams/app/components/AuthorBar.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+    <div class="grid-item-author-bar mt-1">
+        <div @click.stop.prevent="profileBoxTrigger" class="profile-box-trigg">
```

Comment:
Because `profile-box-trigger` is used on the jQuery part. Should I change the `profile-box-trigger` class to `jq-profile-box-trigg`?